src/Engine/ProtoCore/Namespace/ElementResolver.cs

```diff
@@ -1,4 +1,6 @@
+
 ﻿using System.Collections.Generic;
```

Comment:
Just for clarity, would it still mean the same if I said ASTResolver and ASTRewiter?
Im not saying the name should be changed btw.